server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/SearchUsersAction.java

```diff
@@ -19,25 +19,60 @@
  */
 package org.sonar.server.qualityprofile.ws;
 
+import com.google.common.collect.ImmutableMap;
 import java.util.Arrays;
-import org.apache.commons.io.IOUtils;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
 import org.sonar.api.resources.Language;
 import org.sonar.api.resources.Languages;
 import org.sonar.api.server.ws.Request;
 import org.sonar.api.server.ws.Response;
 import org.sonar.api.server.ws.WebService;
+import org.sonar.api.server.ws.WebService.SelectionMode;
+import org.sonar.core.util.stream.MoreCollectors;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.sonar.core.util.stream.MoreCollectors'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)